### ISSUE TITLE
Not compatible with 0.4.0

### ISSUE THREAD

**Issue by vassbence** [LyraSearch/plugin-data-persistence#12]
Currently this plugin is not compatible with Lyra v0.4.0. I think it has to do with outdated types as I get the following error:

```
Property 'docsCount' is missing in type 'Lyra<PropertiesSchema>' but required in type 'Lyra<{...}>'
```

**Comment by paolomainardi** [comment]
Confirmed, it differs also type Data.

**Comment by micheleriva** [comment]
Fixed in `v0.0.5`
